_pages/platform/usage-metrics.md

```diff
@@ -0,0 +1,69 @@
+
+
+# All Usage
+
+This tab shows information grouped by the username/algorithm tuple. That is, it shows which users called which algorithms and the number of times they did so (along with hardware, compute time, and other metrics).
+
+When you click on a row in the all usage table, you will see the complete set of columns:
+- algorithm: algorithm display name
+- version: hash version of the exact algorithm called
+- caller: which user called the algorithm
+- owner: user who created and owns the algorithm
+- total calls: total number of calls the calling user made to this algorithm in the date range
+- total duration: total number of seconds the algorithm ran for caller in the date range
+- total errors: total number of failed calls to this algorithm the caller made in the date range
+- error duration: total number of seconds the algorithm ran for calls in which it returned an error (for this caller)
+- hardware: whether the algorithm runs on a CPU or GPU instance
+
+One caveat to the total compute time: each call is rounded **up** to the nearest second. For instance, if in the given date range, a user calls a certain algorithm twice, and the first call took 2.04 seconds and the second call took 2.08 seconds, the total time for those two calls is 4.12 seconds, which the table will round up and display as 5 seconds. 
+
+Also note that if a call takes less than one second, the call’s duration will be rounded up to a full second. So calling Hello World 23 times would most likely result in a total duration of 23 seconds being displayed. The motivation for this is that in the algorithm marketplace we only charge by credits. One credit = one second, so if a user makes a call that lasts less than a second we still charge for the full second.
+
+
+# Algorithms
+
+This tab shows usage information grouped at the algorithm level. That is, it shows which algorithms were called and how many times (total across all users) they were called.
+
+When you click on a row in the algorithms table, you will see this complete set of columns:
+- algorithm: algorithm display name
+- version: hash version of the exact algorithm called
+- owner: user who created and owns the algorithm
+- total calls: total number of calls that users of the platform made to this algorithm
+- total duration: total number of seconds the algorithm ran summed up across all users
+- total errors: total number of errors this algorithm returned summed up across all users
+- error duration: total number of seconds the algorithm ran calls in which it returned an error (summed up across all users)
+- hardware: whether the algorithm runs on a CPU or GPU instance
+
+
+# Users
```

Comment:
And then update each of these headers to `### All Usage`, `### Algorithms`,` ### Users` so they render as `<h3>`'s.